core/trino-main/src/main/java/io/trino/server/EmbeddedDiscoveryConfig.java

```diff
@@ -17,13 +17,15 @@
 
 public class EmbeddedDiscoveryConfig
 {
-    private boolean enabled;
+    private boolean enabled = true;
 
+    @Deprecated
```

Comment:
This goes beyond "Make .... enabled by default"

at least, add explanation

Reply:
Thanks, I updated the commit message to match the current behavior